src/services/toast-manager/ui/toasts-container/toasts-container.scss

```diff
@@ -1,10 +1,11 @@
 .toastsContainer {
     position: absolute;
+    z-index: 9999;
```

Comment:
i suggest to put a reasonable number, otherwise, we might get into infinite numbers, 
and this is prone to error (the next one will  set 100 and expect to be above all)